ext/semian/resource.h

```diff
@@ -9,6 +9,9 @@ Functions here are associated with rubyland operations.
 #include "types.h"
 #include "sysv_semaphores.h"
 
+// Time to wait for workers to boot using the quota strategy
+#define QUOTA_BOOT_GRACE_PERIOD 60 // seconds
```

Comment:
This is basically around the boot time of the workers, which could, with Ruby on Rails, not unreasonably be 30-90s. I don't know how easy it would be to make this configurable (probably the best thing to do), but I'd say we might need a 90-120s default timeout here for the parameter.